pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 				fmt.Fprintln(o.Out)
 			}
 
+		// path "/" we tried to ping is forbidden, but the server is still reachable
+		case clientcmd.IsForbidden(result.Error()):
```

Comment:
We had osapi before, it broke, then we changed it to /, which broke, so
let's see if we can get it right the third time.

On Mon, Jun 20, 2016 at 10:25 AM, Fabiano Franz <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/loginoptions.go
> <https://github.com/openshift/origin/pull/9413#discussion_r67698383>:
>
> > @@ -159,6 +159,13 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
> >  				fmt.Fprintln(o.Out)
> >  			}
> >
> > +		// path "/" we tried to ping is forbidden, but the server is still reachable
> > +		case clientcmd.IsForbidden(result.Error()):
>
> @liggitt <https://github.com/liggitt> not that I know. If we decide to
> ping something else then this will likely not be needed anymore.
>
> @smarterclayton <https://github.com/smarterclayton> Yeah, it's been here
> forever but I agree there are much better options. Maybe /oapi or /oauth?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9413/files/751e12702dd0ccabd2ee55090978c067dc7e6ff3#r67698383>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_pw5WPTbl2TWNJuGfbMnd_jeoSfvuks5qNqLQgaJpZM4I4yLF>
> .
>